environment.notebooks.yml

```diff
@@ -0,0 +1,30 @@
+name: mapie_notebooks
+channels:
+    - defaults
+    - conda-forge
+dependencies:
+    - bump2version=1.0.1
+    - flake8=3.9.0
+    - ipykernel=6.4.1
+    - jupyter=1.0.0
+    - jupyterlab=3.0.16
+    - jupytext=1.11.2
+    - mypy=0.800
+    - nbconvert=6.4.0
+    - numpydoc=1.1.0
+    - pandas=1.2.4
+    - pandoc=2.17.0.1
+    - pip=21.0.1
+    - pip:
+        - scikeras==0.3.3
+        - tensorflow==2.7.0
+    - pytest=6.2.2
+    - pytest-cov=2.11.1
+    - python=3.9.2
+    - scikit-learn=1.0.0
+    - sphinx=3.5.3
+    - sphinx-gallery=0.8.2
+    - sphinx_rtd_theme=0.5.2
+    - twine=3.4.1
+    - wheel=0.36.2
+    - xgboost=1.3.0
```

Comment:
Could you please remove dependecies not used for creating notebooks ? I think that for example, these packages are useless in this file : 
* bump2version
* flake8
* mypy
* numpydoc
* pytest
* pytest-cov
* sphinx
* sphinx-gallery
* sphinx_rtd_thme
* twine
* wheels

What is `pandoc` used for ?